cf/rds.yaml

```diff
@@ -57,8 +57,8 @@ Resources:
   DBInstance1:
     Type: AWS::RDS::DBInstance
     Properties:
-      DBInstanceClass: db.t3.medium
+      DBInstanceClass: db.t4g.medium
```

Comment:
why this change?

Reply:
db.t4g.medium is the smallest instance that we can use with Postgres 13.4. It is like the t3, but [a little bit cheaper](https://aws.amazon.com/rds/postgresql/pricing/). Actually this instance is not recommended for [non-production use](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.DBInstanceClass.html).

The definition in this file governs the instance for both production and staging. Maybe we should use a bigger instance for production?